diff --git a/main/NekoGui.cpp b/main/NekoGui.cpp
index 2750b3a..776452b 100644
--- a/main/NekoGui.cpp
+++ b/main/NekoGui.cpp
@@ -433,7 +433,7 @@ namespace NekoGui {
     }
 
     QString FindNekoBoxCoreRealPath() {
-        auto fn = QApplication::applicationDirPath() + "/nekobox_core";
+        auto fn = "/run/wrappers/bin/nekobox_core";
         auto fi = QFileInfo(fn);
         if (fi.isSymLink()) return fi.symLinkTarget();
         return fn;
